src/pages/equipo.js

```diff
@@ -1,386 +1,413 @@
+                      />
                     </Box>
-                    <Box className="doubleSpacingBox">
-                        <Box className="spacingBox">
-                            <h2>Infraestructura y redes</h2>
-                        </Box>
-                        <Box>
-                            <Grid container >
-                                {InfrastructureNetworkingTeam.map((person) => (
-                                    <Grid item xs={12} md={6} key={person.name}>
-                                        <Box className="teamPadding" style={{backgroundColor:isMobile? person.color: ""}}>
-                                            <PersonCard 
-                                                img={useBaseUrl(person.img)}
-                                                name={person.name}
-                                                position={person.position}
-                                                bio={person.bio}
-                                                linkedIn={person.linkedIn}
-                                                twitter={person.twitter}
-                                            />
-                                        </Box>
-                                    </Grid>
-                                ))}
-                            </Grid>
-                        </Box>
-                    </Box>     
-                </Box>
+                  </Grid>
+                ))}
+              </Grid>
             </Box>
-        )
-    }
-
-    const SendCV = () => {
-    
-        return (
-            <Box className="containerGray">
-                <Box className="sectionPadding">
-                    <Grid container spacing={5}>
-                        <Grid item xs={12} md={8}> 
-                            <Box>
-                                <h2 className="centerTextOnMobile">¿Le interesa trabajar con nosotros? </h2>
-                            </Box>
-                        </Grid>
-                        <Grid item xs={12} md={4}>
-                            <Box className='centerTextOnMobile'>
-                                <button className="buttonPrimary" onClick={() => window.open(`mailto:${'jobs@eoscostarica.io'}?subject=${'Mi CV - quiero trabajar para tu empresa'}`)} >Envíe su CV</button>
-                            </Box>
-                        </Grid>
-                    </Grid>
-                </Box>
+          </Box>
+          <Box className="doubleSpacingBox">
+            <Box className="spacingBox">
+              <h2>Infraestructura y redes</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {InfrastructureNetworkingTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```

Comment:
Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop